ispcrt/CMakeLists.txt

```diff
@@ -52,158 +52,103 @@ if (ISPCRT_BUILD_GPU)
-    target_compile_definitions(${TARGET_NAME} PRIVATE ISPCRT_BUILD_GPU)
-  endif()
+## Install targets + exports ##
 
-  if (ISPCRT_BUILD_CPU)
-    target_compile_definitions(${TARGET_NAME} PRIVATE ISPCRT_BUILD_CPU)
-  endif()
+set_target_properties(${PROJECT_NAME}
+    PROPERTIES VERSION ${PROJECT_VERSION} SOVERSION ${PROJECT_VERSION_MAJOR})
 
-  target_include_directories(${TARGET_NAME}
-  PUBLIC
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/../common>
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}>
-    $<BUILD_INTERFACE:${LEVEL_ZERO_INCLUDE_DIR}>
-    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/ispcrt>
-  )
-
-  target_link_libraries(${TARGET_NAME}
-  PRIVATE
-    $<$<BOOL:${ISPCRT_BUILD_GPU}>:${LEVEL_ZERO_LIB_LOADER}>
-    ${CMAKE_DL_LIBS}
-    $<$<BOOL:${ISPCRT_BUILD_TASKING}>:ispcrt_tasking>
-  )
-
-  # Security options
-  if (MSVC)
-    # Stack canaries
-    target_compile_options(${TARGET_NAME} PRIVATE /GS)
-    # Control flow guard
-    target_link_options(${TARGET_NAME} PRIVATE /GUARD:CF)
-  elseif (APPLE)
-  else()
-    # Enable stack protector
-    # NOT to assume that null pointer deference does not exist
-    # Assume that signed overflow always wraps
-    target_compile_options(${TARGET_NAME} PRIVATE -fstack-protector-strong -fno-delete-null-pointer-checks -fwrapv)
-  endif()
+install(TARGETS ${PROJECT_NAME}
+  EXPORT ${PROJECT_NAME}_Exports
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    NAMELINK_SKIP
+  # on Windows put the dlls into bin
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  # ... and the import lib into the devel package
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
 
-  ## Install targets + exports ##
-
-  set_target_properties(${TARGET_NAME}
-      PROPERTIES VERSION ${PROJECT_VERSION} SOVERSION ${PROJECT_VERSION_MAJOR})
-
-  install(TARGETS ${TARGET_NAME}
-    EXPORT ${PROJECT_NAME}_Exports
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-      NAMELINK_SKIP
-    # on Windows put the dlls into bin
-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-    # ... and the import lib into the devel package
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  )
-
-  install(EXPORT ${PROJECT_NAME}_Exports
-    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}-${PROJECT_VERSION}
-    NAMESPACE ${PROJECT_NAME}::
-  )
-
-  install(TARGETS ${TARGET_NAME}
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-      NAMELINK_ONLY
-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  )
-endmacro()
+install(EXPORT ${PROJECT_NAME}_Exports
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}-${PROJECT_VERSION}
+  NAMESPACE ${PROJECT_NAME}::
+)
+
+install(TARGETS ${PROJECT_NAME}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    NAMELINK_ONLY
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
 
-if (WIN32)
-    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
-endif()
-build_ispcrt(SHARED ${PROJECT_NAME})
 if (WIN32)
```

Comment:
Why spreading ON/OFF pair for `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` between this CMakeList and includes?